src/Context.fs

```diff
@@ -47,6 +47,8 @@ and HttpRequest = {
     Logger: ILogger option
     /// The LogLevel to log at
     LogLevel: LogLevel
+    /// Logging format string
```

Comment:
I would be nice to include which parameters are actually available in the format string.